react/src/views/Pages/LoginPage.jsx

```diff
@@ -42,19 +45,36 @@ class LoginPage extends React.Component {
   }
 
   handleLogin = () => {
+    // this.setState(
+    //   {
+    //     open_loginFail: true,
+    //   }
+    // )
```

Comment:
This code shouldn't be here. It's causing the `Login Fail` popup to show even when the user is successfully authenticated.

Reply:
line 48-52 comment out